src/components/AgentSetupBuilder/styles.module.css

```
@@ -0,0 +1,146 @@
+  color: var(--ifm-color-emphasis-600);
+}
+
+.planeReqBadge {
+  font-size: 0.65rem;
+  color: var(--ifm-color-emphasis-500);
+  font-style: italic;
+}
+
+.builder :global(.prism-code) {
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)